.github/workflows/publish-to-pypi.yaml

```
@@ -84,175 +99,377 @@ jobs:
           compiler: gcc
           version: 13
-        # This sets up gfortran and gcc for macOS runners
-        # FC, CC, and CXX environment variables will be set for subsequent steps
-        # See https://github.com/fortran-lang/setup-fortran for details
+          # This sets up gfortran and gcc for macOS runners
+          # FC, CC, and CXX environment variables will be set for subsequent steps
+          # See https://github.com/fortran-lang/setup-fortran for details
       - name: Build x86_64 wheels
-        uses: pypa/cibuildwheel@v2.21.3
+        uses: pypa/cibuildwheel@v3.1.3
+        if: ${{ false && github.event.inputs.build_legacy_wheels }} # FIXME: Unable to build x86_64 wheels on macOS due to gfortran architecture being arm64
```

actionlint (1.7.9)

property "build_legacy_wheels" is not defined in object type {build_aarch64: string; build_i686: string; build_legacy: string; build_manylinux: string; build_musllinux: string; build_wheels_macos: string; build_windows: string; pypi_repo: string; version: string}

(expression)